src/CMakeProject-texture.cmake

```diff
@@ -37,10 +37,6 @@ set_property(TARGET "TextureFontGenerator" PROPERTY FOLDER "Internal Libraries")
 
 disable_project_warnings("TextureFontGenerator")
 
-if(NOT WITH_STATIC_LINKING)
-  target_compile_definitions("TextureFontGenerator" PRIVATE _AFXDLL)
```

Comment:
Does this mean we're not compiling the TextureFontGenerator?

Reply:
We do compile the TextureFontGenerator. It just doesn't need the `_AFXDLL` define because I switched the binaries to the static MSVC runtime, so users don't need to install the VC runtime separately.